platform/ext/common/gcc/tfm_common_s.ld

```diff
@@ -1412,6 +1358,36 @@ SECTIONS
     } > RAM AT> FLASH
 #endif
 
+#ifndef TFM_MULTI_CORE_TOPOLOGY
+    /*
+     * Place the CMSE Veneers (containing the SG instructions) in a correctly
+     * aligned region so that the SAU can programmed to just set this region as
```

Comment:
SAU or SPU?

Reply:
This is the common file. I can change it to "SAU/IDAU".
```suggestion
     * aligned region so that the SAU/IDAU can programmed to just set this region as
```